src/pages/Provider/Provider.stories.js

```diff
@@ -0,0 +1,50 @@
+import ProviderLayout from './Provider.layout.vue';
+import ProviderDoc from './Provider.documentation.mdx';
+import { providers } from '../Providers/mock';
+import { providerProducts } from './components/ProviderProducts/mock';
+
+export default {
+  title: 'Pages/Provider',
+  component: ProviderLayout,
+  parameters: {
+    docs: {
+      page: ProviderDoc,
+    },
+  },
+  argTypes: {
+    provider: {
+      name: 'provider',
+      type: {
+        required: true,
+      },
+      description: 'The provider',
+      control: {
+        type: null,
+      },
+    },
+    id: {
+      control: {
+        type: null,
+      },
+    },
+  },
+};
+
+const Template = (args) => ({
+  components: { ProviderLayout },
+  setup() {
+    return { args };
+  },
+  template: '<provider-layout v-bind="args" />',
+});
+
+const currentProvider = providers[0];
+
+export const Page = Template.bind({});
+Page.args = {
+  provider: currentProvider,
+  products: providerProducts,
+  searchText: '',
+  id: currentProvider.id,
```

Comment:
Maybe it needs to be required? Or you can not annotate that here?

Reply:
Yes, I can annotate that here. I will apply that